DNN Platform/DotNetNuke.Web.Client/ClientResourceSettings.cs

```diff
@@ -90,10 +90,7 @@ private UpgradeStatus Status
             }
         }
 
-        /// <summary>
-        ///
-        /// </summary>
-        /// <returns></returns>
+        [Obsolete("Consider using the overload that takes the portalId. That version works also outside the HttpContext")]

```

Comment:
chore: use standard deprecation message

See https://github.com/dnnsoftware/Dnn.Platform/blob/2d257916c458d0c7383c94b3cd7aefe11e3bc4ad/.github/VERSIONING_POLICY.md#api-deprecation-policy-updated-september-2020

```suggestion
        [Obsolete("Deprecated in Platform 9.11.0. Use overload taking portalId. Scheduled removal in v11.0.0.")]
```